### DISCUSSION TITLE
nginx-opentracing why do not support skywalking

### DISCUSSION THREAD

**Discussion by jxplus** [apache/skywalking#11512] [Q&A] [answered]
In nginx opentracing repo[https://github.com/opentracing-contrib/nginx-opentracing], 
I see the second dependency is:
A C++ OpenTracing Tracer. It currently works with Jaeger, Zipkin, LightStep, or Datadog.

means, nginx-opentracing depend a tracer, bug zhe tracer no skywalking impl.

why do not support skywalking, what's the thinking about?

**Comment by wu-sheng** [comment]
Why do you submit duplicate things? It is annoying.

**Reply by wu-sheng** [reply, marked as answer]
Replied here, https://github.com/apache/skywalking/issues/11513

**Reply by jxplus** [reply]
never use discuss，try once ~_~
